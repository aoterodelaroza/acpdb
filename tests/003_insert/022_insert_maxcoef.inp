## check: 022_insert_maxcoef.out -a1e-10
## delete: 022_insert_maxcoef.db
## labels: regression quick

system rm -f 022_insert_maxcoef.db
connect 022_insert_maxcoef.db

insert method m_empty
end
insert method m_ref
end
insert set 022_insert_maxcoef-set
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method m_ref
end
insert evaluation
 method m_empty
 property 022_insert_maxcoef-set.nh3_nh3
 value 0.2
end
insert evaluation
 method m_empty
 property 022_insert_maxcoef-set.h2o_h2o
 value 0.3
end

training
 atom H l
 exp 0.1 0.2
 empty m_empty
 reference m_ref
 subset
  set 022_insert_maxcoef-set
 end
end

insert calc
 property_type energy_difference
 file ../zz_source/terms3.dat
 method m_empty
 term
end
insert maxcoef
 method m_empty
 file ../zz_source/terms3_maxcoef.dat
end
