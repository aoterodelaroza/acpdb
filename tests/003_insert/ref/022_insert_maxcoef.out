* SYSTEM: rm -f 022_insert_maxcoef.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 022_insert_maxcoef.db
Creating skeleton database 

* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_empty

* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_ref

* INSERT: insert data into the database (SET)
# INSERT SET 022_insert_maxcoef-set
# INSERT STRUCTURE 022_insert_maxcoef-set.nh3_nh3
# INSERT STRUCTURE 022_insert_maxcoef-set.nh3_nh3_1
# INSERT STRUCTURE 022_insert_maxcoef-set.nh3_nh3_2
# INSERT PROPERTY 022_insert_maxcoef-set.nh3_nh3
# INSERT EVALUATION (method=m_ref;property=022_insert_maxcoef-set.nh3_nh3)
# INSERT STRUCTURE 022_insert_maxcoef-set.h2o_h2o
# INSERT STRUCTURE 022_insert_maxcoef-set.h2o_h2o_1
# INSERT STRUCTURE 022_insert_maxcoef-set.h2o_h2o_2
# INSERT PROPERTY 022_insert_maxcoef-set.h2o_h2o
# INSERT EVALUATION (method=m_ref;property=022_insert_maxcoef-set.h2o_h2o)

* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=022_insert_maxcoef-set.nh3_nh3)

* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=022_insert_maxcoef-set.h2o_h2o)

* TRAINING: started defining the training set 

* TRAINING: fininshed defining the training set 

## Description of the training set
# List of atoms and maximum angular momentum channels (1)
| Atom | lmax |
| H | l |

# List of exponents (2)
| id | exp |
| 0 | 0.1 |
| 1 | 0.2 |

# List of subsets (1)
| id | alias | db-name | db-id | initial | final | size | dofit? | litref | description |
| 0 | 022_insert_maxcoef-set | 022_insert_maxcoef-set | 1 | 1 | 2 | 2 | 1 |  |  |

# List of methods
| type | name | id | for fit? |
| reference | m_ref | 2 | n/a |
| empty | m_empty | 1 | n/a |

# List of properties (2)
| fit? | id | property | propid | alias | db-set | proptype | nstruct | weight | refvalue |
| yes | 1 | 022_insert_maxcoef-set.nh3_nh3 | 1 | 022_insert_maxcoef-set | 022_insert_maxcoef-set | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 2 | 022_insert_maxcoef-set.h2o_h2o | 2 | 022_insert_maxcoef-set | 022_insert_maxcoef-set | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |

# Calculation completion for the current training set
# Reference: 2/2 (complete)
# Empty: 2/2 (complete)
# Terms: 
| H | l | 0.1 | 0/2 | (missing)
| H | l | 0.2 | 0/2 | (missing)
# Total terms: 0/4 (missing)

* INSERT: insert data into the database (CALC)
# INSERT TERM (method=m_empty;property=2;nvalue=1;atom=1;l=0;exp=0.1)
# INSERT TERM (method=m_empty;property=1;nvalue=1;atom=1;l=0;exp=0.1)
# INSERT TERM (method=m_empty;property=2;nvalue=1;atom=1;l=0;exp=0.2)
# INSERT TERM (method=m_empty;property=1;nvalue=1;atom=1;l=0;exp=0.2)

* INSERT: insert data into the database (MAXCOEF)

* PRINT: print the contents of the database 

| methodid| propid| atom| l| exponent| #values| 1st value| maxcoef|
| 1| 2| 1| 0| 0.1| 1| -156.8773675| 0.35|
| 1| 1| 1| 0| 0.1| 1| -225.9034092| 0.35|
| 1| 2| 1| 0| 0.2| 1| -784.3868375| 1.35|
| 1| 1| 1| 0| 0.2| 1| -853.4128792| 0|

